pages/linux/at.md

```diff
@@ -0,0 +1,11 @@
+# at
+
+> Executes commands at a specified time.
+
+- Open 'at' prompt to execute commands at specific time. When you done press Ctrl+D to register job(s):
+
+`at {{hh:mm:ss}}`
+
+- Execute file (-f) and mail result(-m):
```

Comment:
Will add separate example for mailing option.

I meant to say that the file should be executable script.